port-forward-fun.conf

```diff
@@ -1,19 +0,0 @@
-##################### Notes ########################
-#                                                  #
-#  So, you wanna run a single Node server, huh?    #
-#  Here's what the Apache server might look like   #
-#                                                  #
-#  This will assume an Express server is already   # 
-#                running on Port 3000              #
-#                                                  #
-####################################################
-
-<VirtualHost *:80>
```

Comment:
A VirtualHost is basically what it says on the tin: A host that is virtual!

Since we don't have a single host at Port 80 but multiple, I suppose that's why they call 'em virtual.

This line is saying that we want to create a new VirtualHost that is triggered anytime a request is made at Port 80.

* denotes the IP address (it's a wildcard so any IP address) but since this file is sitting inside the server, it will basically only resolve to be the IP address of the server.

The main reason for this is that if your IP address is not static and changes, you'll have to keep changing the IP address in your log to ensure it can answer requests.